api/app/signals/apps/api/v1/serializers/signal.py

```diff
@@ -285,7 +285,7 @@ def create(self, validated_data):
 class PublicSignalSerializerDetail(HALSerializer):
     status = _NestedPublicStatusModelSerializer(required=False)
     serializer_url_field = PublicSignalLinksField
-    _display = DisplayField()
+    _display = serializers.SerializerMethodField(method_name='get__display')
```

Comment:
The `method_name` is not necessary, the `SerializerMethodField` looks for a `get_{field_name}` method.

Maybe this could be in the `__str__` of the Signal model?